src/core_plugins/kibana/public/dashboard/top_nav/clone_modal.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  KuiModal,
+  KuiModalHeader,
+  KuiModalHeaderTitle,
+  KuiModalBody,
+  KuiModalBodyText,
+  KuiModalFooter,
+  KuiButton,
+  KuiModalOverlay
+} from 'ui_framework/components';
+
+export class DashboardCloneModal extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      newDashboardName: props.title
+    };
+  }
+
+  componentDidMount() {
+    this.confirmButton.focus();
```

Comment:
I didn't use autoFocus because it's not supported by all browsers here: http://caniuse.com/#feat=autofocus

Though honestly, I thought it wasn't supported  on Safari, but a closer look reveals it's "IOS" safari, so the phone browser, and Opera Mini.  So maybe it's okay?  Since they are touch screen devices, you wouldn't be pressing enter anyway, but touching one of the buttons.

@epixa, do we have a set of browsers we have to support?